config/default.yml

```diff
@@ -923,6 +923,28 @@ RSpec/FactoryBot:
   Include: *1
   Language: *2
 
+RSpec/FactoryBot/AssociationStyle:
+  Description: Use consistent style to define associations.
+  Enabled: pending
+  Safe: false
+  inherit_mode:
+    merge:
+      - NonImplicitAssociationMethodNames
+  Include:
+    - spec/factories.rb
+    - spec/factories/**/*.rb
+    - features/support/factories/**/*.rb
+  VersionAdded: "<<next>>"
+  EnforcedStyle: implicit
+  SupportedStyles:
+    - explicit
+    - implicit
+  NonImplicitAssociationMethodNames:
+    - association
+    - skip_create
+    - traits_for_enum
```

Comment:
Does this have to be user configurable?

Reply:
Certainly there is no need to make these default values removable. The main reason why I have this config value is for [Global sequences](https://github.com/thoughtbot/factory_bot/blob/5c01f05/GETTING_STARTED.md#global-sequences).

```ruby
Factory.define do
  sequence(:email) { |n| "person#{n}@example.com" }

  factory :user do
    email
  end
end
```